artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -726,6 +744,25 @@ public int getAddressMemoryUsagePercentage() {
       return (int) result;
    }
 
+   @Override
+   public int getDiskUsedStorePercentage() {
+      if (AuditLogger.isEnabled()) {
+         AuditLogger.getDiskStoreUsagePercentage(this.server);
+      }
+      long globalMaxSize = getGlobalMaxSize();
+      // no max size set implies 0% used
+      if (globalMaxSize <= 0) {
+         return 0;
+      }
+
+      long diskUsed = getDiskStoreUsed();
+      if (diskUsed <= 0) {
+         return 0;
+      }
+      double result = (100D * diskUsed) / globalMaxSize;
```

Comment:
You should use `org.apache.activemq.artemis.core.server.files.FileStoreMonitor#calculateUsage` here.